examples/finite_temperature/metts.jl

```diff
@@ -1,5 +1,6 @@
 using ITensors, ITensorMPS
 using Printf
+using Random: rand
```

Comment:
I don't think this is needed since `rand` is defined in and exported from Base:
```julia
julia> @which rand
Base

julia> Base.isexported(Base, :rand)
true
```

Reply:
Good catch – removed